### ISSUE TITLE
Cannot redefine non-configurable property 'values' on <= IE11

### ISSUE THREAD

**Issue by k1sul1** [Financial-Times/polyfill-service#1341]
I've included this before my bundle:
```
<script type='text/javascript' src='https://cdn.polyfill.io/v2/polyfill.min.js?features=default%2Ces6%2Cfetch&#038;ver=4.8.2'></script>
```

However, IE11 (and lower) crash on an error: 
![image](https://user-images.githubusercontent.com/2719615/30710244-9e930e98-9f0d-11e7-8b8a-8a4ed6583b8e.png)

**Comment by DonGissel** [comment]
You should probably try again with the unminified version, in order to locate the actual place it crashes :-)

**Comment by JakeChampion** [comment]
@k1sul1 Please could you create a small reproducible test case using jsbin.com. Without that we will struggle to help with the issue.

**Comment by k1sul1** [comment]
Sure. I'll try to find time to create one tomorrow.

I've had this happen on multiple WordPress sites that I've built during the last year, some of them with Rollup + Bublé, and the latest one with Webpack + Babel. 

If I'm unable to reproduce this using jsbin.com, it will probably relate to my builds or WordPress.

**Comment by k1sul1** [comment]
Doesn't reproduce on jsbin.com

http://jsbin.com/zibupozoli/edit?html,css,output

🤔

**Comment by ipanasenko** [comment]
Hey guys. I've just faced the same issue.

It work fine for me on non-minified version, but I get the same error in .min.js version.
This seems to be coming from `es6` feature

**Comment by falconmick** [comment]
Having issue when I use this:
<script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=default,es6"></script>

**Comment by sebastianekstrom** [comment]
Experiencing the same issues in IE11 where the minified build doesn't work.

This works fine:
https://cdn.polyfill.io/v2/polyfill.js?features=es5,es6,es2016,es2017|gated&flags=gated&unknown=polyfill&callback=onPolyfillsLoad

This crashes:
https://cdn.polyfill.io/v2/polyfill.min.js?features=es5,es6,es2016,es2017|gated&flags=gated&unknown=polyfill&callback=onPolyfillsLoad

with the error `Cannot redefine non-configurable property 'values'` in IE11

**Comment by hjmccain** [comment]
hey @k1sul1, did you end up finding a fix for this? i built a small app to sit inside of a drupal website, and i am running into the same issue you did. when i try the unminified version i also get an error ("Cannot define property '__st59989263179437283__': object is not extensible.")

**Comment by k1sul1** [comment]
Nope. I recalled one site that I made and used polyfill.io, and it seems to be working. 

I've since used Webpack & babel-polyfill for polyfilling.

**Comment by aaarichter** [comment]
MDN has a page for this error: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Cant_redefine_property

The issue occurs in the Symbol polyfill, which is a dependency to the Promise polyfill: https://github.com/Financial-Times/polyfill-service/blob/8717a9e04ac7aff99b4980fbedead98036b0929a/packages/polyfill-library/polyfills/Symbol/polyfill.js#L222-L229

screenshot:
<img width="1059" alt="screen shot 2018-04-29 at 1 34 19 pm" src="https://user-images.githubusercontent.com/2707029/39410297-20308240-4bbc-11e8-8d0b-1a337d136a47.png">
`defineGetter`: https://github.com/Financial-Times/polyfill-service/blob/8717a9e04ac7aff99b4980fbedead98036b0929a/packages/polyfill-library/polyfills/_DOMTokenList/polyfill.js#L10-L18
`defineProp`: https://github.com/Financial-Times/polyfill-service/blob/8717a9e04ac7aff99b4980fbedead98036b0929a/packages/polyfill-library/polyfills/Symbol/polyfill.js#L127-L137
`setDescriptor`: https://github.com/Financial-Times/polyfill-service/blob/8717a9e04ac7aff99b4980fbedead98036b0929a/packages/polyfill-library/polyfills/Symbol/polyfill.js#L222-L229

**Comment by aaarichter** [comment]
the problem is within the symbols polyfill

**Comment by falconmick** [comment]
any chance of an ETA for a fix for this? All of my sites have been forced to use un-minified version. I would submit a PR myself but I have 0 clue how to even get started with this.

**Comment by horyd** [comment]
I just ran into this error, including on the un-minifed version. Am using babel-polyfill in the interim.

**Comment by simkessy** [comment]
...

**Comment by luke-dare** [comment]
Experiencing the same problem here ...
